src/Tizen.Content.Download/Tizen.Content.Download/Request.cs

```diff
@@ -843,6 +844,213 @@ private void UnregisterProgressChangedEvent()
                 DownloadErrorFactory.ThrowException(ret, "Unsetting ProgressChanged callback failed");
             }
         }
+
+        /// <summary>
+        /// Sets the 'enabled' state of the cache feature.
+        /// </summary>
+        /// <since_tizen> 7.5 </since_tizen>
+        /// <param name="enable">The enable value</param>
+        /// <privilege>http://tizen.org/privilege/download</privilege>
+        /// <remarks>
+        /// Enable/disable for _downloadId
+        /// </remarks>
+        /// <exception cref="ArgumentException">Thrown when it is failed due to an invalid parameter.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when it is failed due to an invalid operation.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when a permission is denied.</exception>
+        /// <exception cref="NotSupportedException">Thrown when not supported.</exception>
+        public void SetCacheEnableState(bool enable)
+        {
+            int ret = Interop.Download.SetDownloadCache(_downloadId, enable);
+            if (ret != (int)DownloadError.None)
+            {
+                DownloadErrorFactory.ThrowException(ret, "Failed to set enabled state of download cache");
+            }
+        }
+
+        /// <summary>
+        /// Returns the 'enabled' state of the cache feature.
+        /// </summary>
+        /// <since_tizen> 7.5 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/download</privilege>
+        /// <remarks>
+        /// Get Enable/disable for _downloadId
```

Comment:
'_downloadId' is a private value. It's not appropriate to show in the API level.

Reply:
For set/get APIs, how about using C# Property? Please refer to 'public string DestinationPath {get; set;}'

Reply:
Changed methods to C# Properties.